test.sh

```diff
@@ -1,11 +1,15 @@
-{
-	  mv go.mod1 go.mod
-	  mv go.sum1 go.sum
-	  GO111MODULE=on go test -p 1 -race -covermode=atomic -coverprofile=coverage.txt -coverpkg=./... ./...
-} || {
-	  mv go.mod go.mod1
-	  mv go.sum go.sum1
-}
+#!/usr/bin/env bash
 
-mv go.mod go.mod1
-mv go.sum go.sum1
+exitCode=0
```

Comment:
Please just use `set -e`. 

In the current implementation, if `go test` failed the shell script would still `exit 0`.

Reply:
But the command `mv go.mod go.mod1 && mv go.sum go.sum1` should be executed regardless of the test succeeds or fails.

Reply:
@tangenta 

```sh
mv go.mod go.mod1 && mv go.sum go.sum1 || true
```

Reply:
@tangenta You could also use `trap EXIT`, or retain the previous code structure.